src/rgw/driver/rados/rgw_data_sync.cc

```diff
@@ -2122,7 +2121,7 @@ class RGWDataIncSyncShardCR : public RGWDataBaseSyncShardCR {
       } while (true);
 
       drain_all();
-      yield marker_tracker->flush();
+      yield call(marker_tracker->flush());
```

Comment:
yikes, good catch. without the `call()`, nothing would actually run that coroutine, and its memory would just leak